usr/share/rear/output/ISO/Linux-i386/250_populate_efibootimg.sh

```diff
@@ -28,7 +28,7 @@ if test -f "$SECURE_BOOT_BOOTLOADER" ; then
     # (cf. rescue/default/850_save_sysfs_uefi_vars.sh)
     # then Shim (usually shim.efi) was copied above as efi_boot_tmp_dir/BOOTX64.efi
     # and Shim's second stage bootloader must be also copied where Shim already is.
-    DebugPrint "Using Shim '$SECURE_BOOT_BOOTLOADER' as first stage UEFI bootloader BOOTX64.efi"
+    DebugPrint "Using Shim '$SECURE_BOOT_BOOTLOADER' as first stage UEFI bootloader BOOT${EFI_ARCH_UPPER}.efi"
```

Comment:
I think ${VAR} is not needed here so
```
BOOT$EFI_ARCH_UPPER.efi
```
is sufficient and that is used some lines above in
```
cp ... $efi_boot_tmp_dir/BOOT$EFI_ARCH_UPPER.efi ...
```

Reasoning:

I get on command line with openSUSE Leap 15.5
```
# foo=foo

# bar=bar$foo.baz

# echo "'$bar'"
'barfoo.baz'
```

This matches "man bash" (for bash version 4.4.23)
```
DEFINITIONS
...
  name  A word consisting only of alphanumeric characters
        and underscores, and beginning with an alphabetic
        character or an underscore. Also referred to as an
        identifier.
...
PARAMETERS
...
  A variable may be assigned to by a statement of the form
    name=[value]
```

So all non-alphanumeric characters and non-underscores
should be delimiters of variable names.

Reply:
> I think ${VAR} is not needed here so
> 
> ```
> BOOT$EFI_ARCH_UPPER.efi
> ```
> 
> is sufficient

It is certainly not needed for the shell, but I find that variable expansion in the middle of a longer string is hard to read without the curly braces. My judgement of what is hard enough to read and what is not yet is subjective, hence some inconsistency in the usage of the curly braces.

Reply:
@pcahyna 
this inconsistency is perfectly fine with me,
see my comment below
https://github.com/rear/rear/pull/3157#pullrequestreview-1914753506

Reply:
But shouldn't then for a particular `prefix$VARIABLE.suffix`
one of both forms `prefix$VARIABLE.suffix`
or `prefix${VARIABLE}.suffix`
be used consistently everywhere in the code
to have same readability everywhere in the code?

Reply:
I'm for

1. concise
2. legible

variable names. So by default short without braces but in places where the eye has a hard time reading better with braces.